Birthday-2024-Project/Scripts/Game/States/GameState.gd

```diff
@@ -3,22 +3,53 @@ extends Node2D
 
 var manager: GameManager
 
-
-func enter_state():
-	pass
-
+func next_puzzle(skipConfirm : bool = true):
```

Comment:
it will always be true, even with Win State achieved

Reply:
The idea is that the next button on the win screen will provide false to that argument. Though, I realized that skip button is still accessible on the win screen, so I overrided the function on win state to skip confirmation. I don't know if there'll be a case to not have the confirmation, but the ability is there